lib/hg-utils.coffee

```diff
@@ -242,7 +242,11 @@ class Repository
       params = []
     if !util.isArray(params)
       params = [params]
-    child = spawnSync('hg', params)
+
+    return '' unless @isCommandForRepo(params)
+
+    options = { cwd: @rootPath }
+    child = spawnSync('hg', params, options)
```

Comment:
Could you try it again? I'm almost sure it was caused by the 'restart' needing after installing mercurial :P